src/command_modules/azure-cli-deploymentmanager/azure/cli/command_modules/deploymentmanager/_help.py

```diff
@@ -0,0 +1,198 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['deploymentmanager'] = """
+    type: group
+    short-summary: Create and manage rollouts for your service.
+    long-summary: To deploy your service across many regions and make sure it is running as expected in each region, you can use deployment manager to coordinate a staged rollout of the service. For more details, visit https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-manager-overview
+"""
+
+helps['deploymentmanager artifact-source create'] = """
+    type: command
+    short-summary: Creates an artifact source.
+    examples:
+        - name: Create a new artifact source.
+          text: az deploymentmanager artifact-source create -g rg1 --artifact-source-name contosoServiceArtifactSource -l centralus --sas-uri https://dummy.blob.azure.com/samplesasuri
+"""
+
+helps['deploymentmanager artifact-source show'] = """
+    type: command
+    short-summary: Get the details of an artifact source.
+    examples:
+        - name: Get an artifact source
+          text: >
+            az deploymentmanager artifact-source show -g rg1 --artifact-source-name contosoServiceArtifactSource
+"""
+
+helps['deploymentmanager artifact-source delete'] = """
+    type: command
+    short-summary: Deletes an artifact source.
+    examples:
+        - name: Deletes an artifact source
+          text: >
+            az deploymentmanager artifact-source delete -g rg1 --artifact-source-name contosoServiceArtifactSource
```

Comment:
`--artifact-source-name` should be aliased to `--name/-n`

Reply:
It is. Should the samples use the alias or the actual param name?

Reply:
`--artifact-source-name` should be `--name`

Reply:
Leaving the long name as well after our discussion. However, updated all examples to use '-n' and not the long name.